.cookiecutter/includes/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+name = "h-testkit"
+dynamic = ["description", "version", "urls", "requires-python", "readme", "dependencies"]
+classifiers = [
+    "Framework :: Pytest",
+]
+
+[project.entry-points.pytest11]
+myproject = "h_testkit.pytest_plugin"
```

Comment:
Tell pytest about h-testkit's pytest plugin. This means that whenever h-testkit is installed as a test requirement in a project's tests then h-testkit's pytest plugin will be activated.